frontend/src/metabase/components/ItemsTable/BaseItemsTable/BaseItemsTable.tsx

```diff
@@ -105,8 +108,10 @@ export type BaseItemsTableProps = {
   selectedItems?: CollectionItem[];
   hasUnselected?: boolean;
   isPinned?: boolean;
-  sortingOptions?: SortingOptions;
-  onSortingOptionsChange?: (newSortingOptions: SortingOptions) => void;
+  sortingOptions?: SortingOptions<ListCollectionItemsSortColumn>;
```

Comment:
`BaseItemsTableProps` - despite its name - is tightly coupled to collection items and is never used in any other context.